internal/services/watchlist/watchlist.go

```diff
@@ -0,0 +1,62 @@
+package watchlist
+
+import (
+	"Tradeasy/config"
+	"Tradeasy/internal/model"
+	"errors"
+)
+
+func CreateWatchlist(CrReq CreateRequest) (CrRes CreateResponse, err error) {
+	var wl model.Watchlist
+	wl.Name = CrReq.WatchlistName
```

Comment:
Why this name is getting filled in watchlist model?

Reply:
done